R/utils.R

```diff
@@ -109,115 +109,6 @@ fuse_sequentially <- function(x, y) {
   c(x, y[sel_names_y])
 }
 
-#' List of `grob` object
-#'
-#' @param ... (`grob`) objects.
-#' @returns a `grob_list` object.
-#' @export
-grob_list <- function(...) {
-  ret <- list(...)
-  assert_list(ret, types = c("grob"))
-  structure(
-    ret,
-    class = c("grob_list", "list")
-  )
-}
-
-# gg_list ----
-
-#' List of `gg` object
-#'
-#' @param ... (`ggplot`) objects.
-#' @returns a `gg_list` object.
-#' @rdname gg_list
-#' @export
-gg_list <- function(...) {
```

Comment:
we may keep the name but use `gg_list <- function(...) {lifeycycle warning; list(...)}` to have some compatibility

Reply:
added back with deprecation warning